vertx-client-shaded/build.gradle

```diff
@@ -83,6 +83,8 @@ shadowJar {
     }
 }
 
+test.dependsOn shadowJar
```

Comment:
I'm curious. Why does it need the shadow jar now but didn't before?

Reply:
```
LibraryTest > Ensures that the shading is correct for the vertx-client-shaded project FAILED
```

It is to fix the flaky test, `LibraryTest`. The test requires shadow jar being built. However, when the test runs, the jar is empty. 

That being said, test already depends on shadowJar. It needs a different fix.